### ISSUE TITLE
The `onRest` doesn't get called when the wheel is stopped by hand (click or tap)

### ISSUE THREAD

**Issue by matronator** [CrazyTim/spin-wheel#44]
I looked at the code and it looks like in the `events.js` the touch/mouse events don't discriminate between a stationary wheel and a spinning one. I think there should be some condition to check if the wheel is spinning and is stopped by the tap/click, to call `onRest`. I would do a PR, but I'm not sure if I have it correct.

**Comment by muhammadbasitobaid** [comment]
Yes, there should be something like onStop for stop() or onRest() should be called for stop as well!!!
